src/test/java/com/exasol/udfdebugging/modules/ExampleUdfSetup.java

```diff
@@ -0,0 +1,57 @@
+package com.exasol.udfdebugging.modules;
+
+import java.io.Closeable;
+import java.sql.*;
+import java.util.stream.Stream;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.containers.ExasolContainer;
+import com.exasol.dbbuilder.dialects.exasol.*;
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript;
+import com.exasol.udfdebugging.ExposedServiceAddress;
+import com.exasol.udfdebugging.LocalServiceExposer;
+
+public class ExampleUdfSetup implements Closeable, AutoCloseable {
```

Comment:
Maybe a small javadoc could be added here? I thought that `example` class is something for users